qutebrowser/misc/sessions.py

```diff
@@ -214,10 +214,18 @@ def _save_tab(self, tab, active):
             data['history'].append(item_data)
         return data
 
-    def _save_all(self):
+    def _save_all(self, only_active_window = False):
         """Get a dict with data for all windows/tabs."""
         data = {'windows': []}
-        for win_id in objreg.window_registry:
+        winlist = objreg.window_registry
+        if only_active_window:
+            active_window = QApplication.instance().activeWindow()
+            winlist = [getattr(active_window, 'win_id', None)]
+            if winlist[0] is None:
+                # Fall back to all windows
+                winlist = objreg.window_registry
```

Comment:
You can get the active window in the command handler (`def session_save`) instead, by having a `win_id` argument and adding a `@cmdutils.argument('win_id', win_id=True)` decorator. Then simply pass something like `only_window=win_id` here.